sdf.md

```diff
@@ -473,6 +474,18 @@ results.
 Actions may also not always complete and may result in application
 errors, such as an item blocking the closing of an automatic door.
 
+One idiom for giving an action initiator status and control about the
+ongoing action is to provide a URI for an ephemeral "action resource"
+in the sdfAction output data, allowing the action both to deliver
+immediate feedback (including errors that prevent the action from
+starting) and to allow the action initiator to use the action resource
```

Comment:
Not sure if I understood the sentence correctly here but I suppose "allowing [...] to allow" was not intended?

```suggestion
in the sdfAction output data, allowing the action to deliver
immediate feedback (including errors that prevent the action from
starting) and the action initiator to use the action resource
```